olive/passes/onnx/quantization.py

```diff
@@ -536,24 +536,24 @@ def _run_for_config(
     ) -> ONNXModel:
         from onnxruntime import __version__ as OrtVersion
 
-        if version.parse(OrtVersion) < version.parse("1.17.0"):
-            raise OlivePassError("MatMul4BitsQuantizer is only supported in onnxruntime >= 1.17.0")
+        if version.parse(OrtVersion) < version.parse("1.16.2"):
+            raise OlivePassError("MatMul4BitsQuantizer is only supported in onnxruntime >= 1.16.2")
 
         from onnxruntime.quantization.matmul_4bits_quantizer import MatMul4BitsQuantizer
 
+        # MatMul4BitsQuantizer sets basicConfig to INFO and prints verbose logs as INFO, suppress them
+        # TODO(jambayk): Use the ort logging severity from the workflow if we expose it as environment variable
```

Comment:
Actually, the ort logging severity only impact the logging in c++ code. when I tried last week, I have to use getLogger().setLevel to change the root logger verbosity to make sure the messages could be logged out.

so, do we need change the root logger level so to output the ORT python messages?

Reply:
Yes, I looked at the ort logging severity only works for c++. 
I will thinking whether we would want to set the same logging level for ort python logging by setting the logger for the ort modules at the same level. 

We don't need to change the root logger. Just need to change the level for the logger for the given name space. 
I don't think it is recommended for packages to do anything to the root logger or use basic config. The ort package should not have done that too.

Reply:
In this week, I have to manually set the root logger level to troubleshoot why the quant_pre_process silently fail(the process exit silently). We need find way to config the ORT python logger level from troubleshooting point of view.

Reply:
This sounds good to me. If the logger is created using `logging.getLogger(__name__)`, it makes it easier for us since we could do `logging.getLogger("onnxruntime").setLevel(...)` to set it for ort. 
Or else, I don't know what else we could do without touching the root logger/

Reply:
Actually, I tried to logging.getLogger("onnxruntime").setLevel(...) but the log is not shown correctly. I don't spend time to investigate the root cause.